conda-forge.yml

```diff
@@ -2,4 +2,4 @@ channel_priority: strict
 azure:
   settings_win:
     pool:
-      vmImage: windows-2019
\ No newline at end of file
+      vmImage: windows-2019
```

Comment:
Oops, before I merge, can you ensure this file is not affected. I think the only difference right now is a newline, but given the large history of this PR, can you rebase this on current `main`? Thanks!

Reply:
> I think the only difference right now is a newline, but given the large history of this PR, can you rebase this on current main? Thanks!

Will do!